app/src/main/java/com/ucm/informatica/spread/Fragments/HistoricalFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.ucm.informatica.spread.Fragments;
+
+import android.os.Bundle;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.ucm.informatica.spread.Activities.MainTabActivity;
+import com.ucm.informatica.spread.Presenter.HistoricalFragmentPresenter;
+import com.ucm.informatica.spread.R;
+import com.ucm.informatica.spread.Utils.CustomRecyclerAdapter;
+import com.ucm.informatica.spread.View.HistoricalFragmentView;
+
+public class HistoricalFragment extends Fragment implements HistoricalFragmentView{
+
+    private HistoricalFragmentPresenter historicalFramentPresenter;
+    private RecyclerView historicalListView;
```

Comment:
Recycler view its a little bit tricky, it works as a list view, so you need to define a layout (space, item format, scroll and so on, ie, visual part), an adapter (data we wanna show) and a decorator (as divider between items, just a simple line).

more info  : [recyler view docu](https://www.androidhive.info/2016/01/android-working-with-recycler-view/)